public/app/config/env/staging.js

```diff
@@ -5,9 +5,10 @@
         [
             'util/container',
             'services/searchService',
-            'services/detailService'
+            'services/detailService',
+            'services/cachingService'
         ],
-        function (container, searchService, detailService) {
+        function (container, searchService, detailService, cachingService) {
```

Comment:
I've used it in staging and uat but not in dev.  It could be easily added to dev as well, temporarily, if performance is an issue or for demo purposes.